### ISSUE TITLE
Repo level Readme updates

### ISSUE THREAD

**Issue by krutarth4** [amosproj/amos2023ws04-pipeline-manager#77]
## User story
1. As a SD I should be able to see whole architecture and how to run it

## Acceptance criteria

- [ ] SD guidelines are added to README.md
- [ ] Execution commands are added for docker compose
- [ ] Sub readme.md in each folder is linked 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been tested
* Feature has been merged into the main branch
* Important parts have been documented in the technology wiki
* All acceptance criteria were met
* Product owner approved features
* Developers agreed to release

**Comment by krutarth4** [comment]
@Elementator @eubec 

This is for the main readme.  I will edit it for the whole structure of the repo
